README.md

```diff
@@ -1 +1,204 @@
-# OpenSLO
\ No newline at end of file
+# OpenSLO
+
+## Table of Contents
+
+- [OpenSLO](#openslo)
+  - [Introduction](#introduction)
+  - [Specification](#specification)
+    - [Goals](#goals)
+    - [Object Types](#object-types)
+      - [General Schema](#general-schema)
+        - [Notes](#notes)
+      - [SLO](#slo)
+        - [Notes](#notes-1)
+        - [Objectives](#objectives)
+      - [Service](#service)
+
+## Introduction
+
+The intent of this document is to outline the OpenSLO specification.
+
+The goal of this project is to provide an open specification for defining and
+interfacing with SLOs to allow for a common, giving a set vendor-agnostic
+solution to defining and tracking SLOs. Platform specific implimentation details
+are purposefully excluded from the scope of this specification.
+
+## Specification
+
+### Goals
+
+- Compliance with the Kubernetes YAML format.
+- Vendor agnostic
```

Comment:
- You dashed it above, so dash it here too (or undash below)
- I'm curious as to why k8s is a specific motivator here? Do you expect this to plug in directly?
- When you say "elsewhere", do you mean to enable forks from day zero? Surely we want the core system to be extensible (and extended) instead?

Reply:
> I'm curious as to why k8s is a specific motivator here? Do you expect this to plug in directly?

 I think that for us using that format checked a lot of boxes.  Moving forward, I don't imagine that we we would have it plug in directly, no

Reply:
fixed

Reply:
> I'm curious as to why k8s is a specific motivator here? Do you expect this to plug in directly?

I would say the goal is to be idiomatic of Kubernetes design, for the same reason @ian-bartholomew describes; a lot of thought/consensus has gone into it, so we can benefit from those design decisions and make the format familiar in structure. We should not constrain ourselves to k8s yaml but take it as inspiration.